### PR TITLE
Multiple search result

### PR DESCRIPTION
**Pull request by borosr**

### CODE REVIEW

File: fireplace.go [hunk 1]
```diff
@@ -126,31 +130,39 @@ func (h *Handler) replaceMultipleInFile(path string, replaces []*ReplaceOpts) er
 	return nil
 }
 
-func (h Handler) searchFile(path, searchFor string) (string, uint64) {
+func (h Handler) searchFile(path, searchFor string) []searchScanResult {
```
Comment (PumpkinSeed) [inline fireplace.go:133]: Can you use the SearchResult here?

**Review by PumpkinSeed — CHANGES_REQUESTED** [review]

File: fireplace.go (hunk 1 above)
Comment (borosr) [reply fireplace.go:133]: Only if I add the fileID to the function parameters.

Comment (borosr) [reply fireplace.go]: Done

Comment (PumpkinSeed) [reply fireplace.go]: Actually not, I think you need to remove the searchScanResult at all. You just set the FileID field later.

**Review by PumpkinSeed — APPROVED** [review]

**Pull request merged**